server/config/config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "development": {
+    "username": "postgres",
+    "password": "profyem001",
+    "database": "CenterManager",
```

Comment:
You did not specify the `port` for the db type. Why is that so?

Reply:
I forgot. I will rectify